paddlecloud/paddlecloud/settings.py

```diff
@@ -323,7 +323,7 @@
 ETCD_IMAGE="quay.io/coreos/etcd:v3.2.1"
 
 # domains that allow notebook to enter
-NOTEBOOK_DOMAINS=["www.paddlepaddle.org"]
+NOTEBOOK_DOMAINS=["cloud.paddlepaddle.org"]
```

Comment:
Thanks very much for contributing!

Thought the `settings.py` in the repo should be a configuring template, so is it better to replace the domain to some template string instead?

Reply:
That's will be absolutely better.